code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+
+import 'styles/movieDetails.css';
+
+export const MovieDetails = ({ apiKey, baseUrl, backdropSize, posterSize }) => {
+  const { movieId } = useParams();
+  const [movieDetails, setMovieDetails] = useState({});
+  const DETAILS_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}`;
+
+  useEffect(() => {
+    fetch(DETAILS_URL)
+    .then(res => res.json())
+    .then((json) => {
+      setMovieDetails(json)
+    })
+  }, [DETAILS_URL]);
+
+  return (
+    <article className='detail-page'>
+      <Link 
+        className='back-link' 
+        to='/' 
+        style={{color: !movieDetails.backdrop_path ? 'black' : 'white'}}
+      >
+          <img src={require('assets/backbutton.svg')} alt=''/>
```

Comment:
`alt` attributes should be descriptive for accessibility reasons - empty strings are not really very helpful :)